src/Microsoft.ApplicationInsights/DataContracts/TelemetryContext.cs

```diff
@@ -132,13 +132,13 @@ public LocationContext Location
             get
             {
                 var result = new Dictionary<string, string>();
-                this.component?.UpdateTags(result);
-                this.device?.UpdateTags(result);
-                this.cloud?.UpdateTags(result);
-                this.session?.UpdateTags(result);
-                this.user?.UpdateTags(result);
-                this.operation?.UpdateTags(result);
-                this.location?.UpdateTags(result);
+                this.Component?.UpdateTags(result);
```

Comment:
Is there a test like you did for `CopyFrom()` that we can do for `UpdateTags()`?